plugins/Marketplace/Controller.php

```diff
@@ -151,6 +151,19 @@ public function manageLicenseKey()
         ));
     }
 
+    public function getPaidPluginsToInstallAtOnceParams()
+    {
+        Piwik::checkUserHasSuperUserAccess();
```

Comment:
Might be good to ensure the correct headers are sent

```suggestion
        Piwik::checkUserHasSuperUserAccess();
        Json::sendHeaderJSON();
```

Reply:
Done :+1: